app/src/main/java/com/github/libretube/services/DownloadService.kt

```diff
@@ -132,6 +136,9 @@ class DownloadService : Service() {
                 .setDestinationUri(destination)
                 .setAllowedOverMetered(true) // Set if download is allowed on Mobile network
                 .setAllowedOverRoaming(true)
+        if (isAudio) {
```

Comment:
I would just use a global variable `isNotificationSet` which is false by default and set it to `true` in that scope after creating the first download request with a notification, and check here whether `isNotificationSet` equals true and decide whether to show the notification accordingly :)

Reply:
Hi @Bnyro , I tried your solution....it works fine. But the problem occurs when I try to cancel the ongoing (video + audio) download, only video downloading gets cancelled because it is visible. The audio downloading remains unaffected because it is **hidden** notification & cannot be cancelled. The audio only & video only works fine 👍
<br/>

https://user-images.githubusercontent.com/57897297/197395923-ffa48cf4-a5d2-4151-a879-7f200ccf20db.mp4

<br/>
I'll need some time to figure out another way :)

Reply:
Oh, damn, didn't think about that ...
We still have the option to just show the video name as title for both downloads which of course isn't the best UX-wise solution butter it'd be better than nothing ig :)
But of course there's still hope that we're finding a better way, I'll let you know if I got any other idea how it could be solved ;)

Reply:
For now, can we show the notifications like this?

* **Audio & Video** <br/>
<img src="https://user-images.githubusercontent.com/57897297/197396912-b15399b3-06b8-4864-b172-065d85050d1f.jpg" width="550px" height="300px" /> 

* **Video** <br/>
<img src="https://user-images.githubusercontent.com/57897297/197396925-e3dea5fe-44f7-430f-a8fc-103fb21e643a.jpg" width="600px" height="300px" /> 

* **Audio** <br/>
<img src="https://user-images.githubusercontent.com/57897297/197396931-cce43f63-cf03-4bee-bedc-988235efc3c4.jpg" width="600px" height="300px" />

Reply:
I'd be okay with that, we can still change it later if we'll ever find a better way to solve this :wink: